### PR TITLE
[DO NOT MERGE] POO UPDATE

### PR DESCRIPTION
**Pull request by Manly-Dwarf**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: Kawaii Big Boss, SatounBroadway, MRTY
add: Added Poo, for real this time!
add: Added numerous types of poo at that.
/:cl:

[why]: 
This is currently untested and has issues, **do not merge!** Maintainers please review.

The Poo Update is finally here (sort of)! Instead of just being a lame emote, poo can both be used as a tool for annoyance, self-defense, and humiliation. Antagonists or special jobs also have their own unique types of poo, which may or may not have different effects.

Here's how it('s supposed to) works at the moment: if you're starving, you can't poo. But if you're not, then you can poo just fine. If you're full/fat, then you'll fart out a "giga" poo which is stronger. Using the emote automatically sets you to starving and drops poo on the ground, effectively slowing you down and making dodging impossible mid-combat, meaning self-defense poo throwing is a last resort and you can't spam poo either. It also functions as a very short stun, meaning that anti-security hijinks ensue. I may add stamina damage in the future just in case poo doesn't have a great enough cost.

Before you complain about poo being unfunny, or immature, let me remind you that you're playing on a server where everyone farts their asses off, specifically on other people's faces, proceeds to eat that ass or uses the fart to propel themselves across space while endlessly screaming. It fits perfectly in with Hippiestation, especially if we're actually going to re-add dildos.

### CODE REVIEW

File: hippiestation/code/game/objects/items/poo.dm [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+		if(H.dna.species.limbs_id == "lizard")
+			poooverlay.icon_state = "poopie_lizard"
+		else
+			poooverlay.icon_state = "poopie_human"
+		//terrible code to support combinations inbound. if you know a way to shorten this please tell me.
+		if(!magic && !bomb && !GIGA) // IF: REGULAR POO
+			H.Knockdown(15) //splat!
+			H.adjust_blurriness(1)
+			H.visible_message("<span class='warning'>[H] is hit in the face by [src]!</span>", "<span class='userdanger'>You've been covered in shit by [src]!</span>")
+		if(!magic && !bomb && GIGA) // IF: GIGA REGULAR POO
```
Comment (yoyobatty) [inline hippiestation/code/game/objects/items/poo.dm:119]: better to use switch cases for these

**Comment by lcass** [comment]
No. Your build failed and this is just ... no.

File: hippiestation/code/game/objects/items/poo.dm (hunk 1 above)
Comment (Manly-Dwarf) [reply hippiestation/code/game/objects/items/poo.dm:119]: ye

**Comment by Manly-Dwarf** [comment]
@lcass what is your argument against it nerd

**Comment by praisenarsie** [comment]
wtf where's the pee

**Comment by Manly-Dwarf** [comment]
argument of the decade: should poo be viable for self-defense (i.e a short stun) or comedic only?

**Comment by lcass** [comment]
We don't need shit for this game to be fun, if it's only there for comedic value then it's cancer and if it's there for actual stun then it's aids. No.

**Comment by Manly-Dwarf** [comment]
You aren't explaining why it's cancer though. Also Carbonhell made a good point about it being used outside of comedy so I'm considering removing the stuns and whatnot.

**Comment by dreamcrab** [comment]
guys the only way this is gonna work is if you do it like i laid out in my original ass update plans

**Comment by dreamcrab** [comment]
contact me on discord and i can explain it for you

**Comment by Manly-Dwarf** [comment]
rekt by teamdad

**Comment by Manly-Dwarf** [comment]
but not dead yet

**Comment by Pyko1** [comment]
No, Teamslab does not know the true function of butt, enact peepee poopoo

**Pull request closed without merging**